packages/block-editor/src/components/link-control/style.scss

```diff
@@ -224,6 +225,10 @@ $preview-image-height: 140px;
 		}
 	}
 
+	.block-editor-link-control__search-item-details {
+		min-width: 0;
+	}
+
```

Comment:
As above, I think we should include a comment to explain what this does and why it's necessary. That would help avoid regressions.